.github/workflows/test.yml

```diff
@@ -0,0 +1,48 @@
+name: Node.js CI
+
+on: [push]
+
+jobs:
+  build-windows:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+        os: [windows-latest]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install dependencies for Viv
+        run: npm install
+      - name: Install dependencies for Avivator
+        run: npm run install:avivator
+      - name: Build Avivator and Viv
+        run: npm run build && npm run build:avivator
+
+  test-linux:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+        os: [ubuntu-latest]
```

Comment:
```suggestion
 test-linux:
    runs-on: ubuntu-latest

    strategy:
      matrix:
        node-version: [14.x]
```

Reply:
I figured leaving it as arrays would make it easier to add future OS if we want modulo a name change but I'm fine iwth less code :)